doc/source/installation.rst

```diff
@@ -0,0 +1,70 @@
+Installation
+============
+
+Depending on your experience, role, computing environment and expectations optimal hardware utilization, you have several ways to install the dependencies required to run PIConGPU.
+Chose your favorite install path below, young padavan:
```

Comment:
REQUIRED:
young padavan (from your demonstration today) is still in here 😄 
Have you ask Micky or George Lucas that it is okay to use this word? 👮